azure-pipelines.yml

```diff
@@ -4,11 +4,18 @@ trigger:
     - '*'
   branches:
     include:
-    - master
+    - '*'
```

Comment:
Why do we need to run builds on each commit in all branches?

Reply:
In my opinion, we should run builds on each commit to master branch + on all pull-requests.